concrete-core/src/backends/core/private/crypto/glwe/tests.rs

```diff
@@ -0,0 +1,100 @@
+use crate::backends::core::private::crypto::encoding::PlaintextList;
```

Comment:
/!\ THIS WHOLE FILE SHOULD BE DELETED

It was only useful before the fixture was introduced. It is now outdated.